hack/ampenv

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
this file is supposed to be sourced, so the shebang will be ignored by the script sourcing this file.